hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -910,7 +910,7 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 
```

Comment:
nit: should probably move this into its own commit.

Reply:
Moved it into a dedicated commit, ptal